client/dfclient/src/gameplay_state_death_reports.cpp

```diff
@@ -0,0 +1,449 @@
+			texts.push_back(theTextCreator->CreateText(
+				"civilian casualty",
+				ncine::Color::Black,
+				textsXPos[0], textsYPos[1]
+				));
+			texts.back()->setPosition(textsXPos[0]+texts.back()->width()/2, textsYPos[1]);
+			texts.push_back(theTextCreator->CreateText(
+				std::to_string(CIVILIAN_PENALTY),
+				ncine::Color::Black,
+				textsXPos[2], textsYPos[1]
+				));
+
+			if (deathReport->multikill() >= 2 || deathReport->killing_spree()) {
+				this->textNodes.push_back(theTextCreator->CreateText(
+					"killing spree stopped",
+					ncine::Color(140, 0, 0),
+					textsXPos[1], textsYPos[0] - killtext->height(),
+					2.0f,
+					255, 30, 70
+					));
+				this->textNodes.push_back(theTextCreator->CreateOutline());
+				this->endKillingSpree();
+			}
+			this->textNodes.push_back(std::move(killtext));
+
+		} else if (deathReport->victim() == (uint16_t)-2) {
+			// it was a GOLDFISH!
+			this->textNodes.push_back(theTextCreator->CreateText(
+				"+1 skill!",
+				ncine::Color(230, 230, 0),
+				textsXPos[1], textsYPos[0],
+				2.5f,
+				255, 30, 60
+				));
+			this->textNodes.push_back(theTextCreator->CreateOutline());
+			_resources.playSound(SoundType::GOLDFISH);	// !
+
+		} else {
+			// it was a PLAYER
+
+			// display "you killed <player>!" and score tally against that player
+			theTextCreator->setTweenParams(255, 45, 90); // timing for "you killed <player>"
+			theTextCreator->setColor(0, 220, 0);
+			auto killtext = theTextCreator->CreateText(
+				"you killed " + gameData.players[deathReport->victim()].name + "!",
+				std::nullopt,
+				textsXPos[1], textsYPos[0],
+				3.0f,
+				std::nullopt, std::nullopt, std::nullopt,
+				std::nullopt,
+				tweeny::easing::sinusoidalIn
+				);
+			auto killtextOutline = theTextCreator->CreateOutline(tweeny::easing::sinusoidalIn);
+
+			auto score = theTextCreator->CreateText(
+				"[" + std::to_string(deathReport->killerKills()) + "  :  " + std::to_string(deathReport->victimKills()) + "]");
+			auto scoreOutline = theTextCreator->CreateOutline();
+			score->setPosition(textsXPos[1], killtext->position().y + killtext->height()/2.f + score->height()/2);
+			scoreOutline->setPosition(score->position().x, score->position().y);
+			
+			this->textNodes.push_back(std::move(killtext));
+			this->textNodes.push_back(std::move(killtextOutline));
+			this->textNodes.push_back(std::move(score));
+			this->textNodes.push_back(std::move(scoreOutline));
+
+			// display score summary: base score and bonuses
+			theTextCreator->setTweenParams(200, 180, 30); // timing for total score
+
+			theTextCreator->setScale(1.2f);
+			theTextCreator->setColor(0, 200, 0);
+			texts.push_back(theTextCreator->CreateText("enemy killed"));
+			texts.push_back(theTextCreator->CreateOutline());
+
+			texts.push_back(theTextCreator->CreateText(std::to_string(KILL_REWARD)));
+			texts.push_back(theTextCreator->CreateOutline());
+
+			int totalScore = KILL_REWARD;
+			int scoreBonus;
```

Comment:
Let's avoid uninitialized variables. Set it to `0` here, or even better, declare it in each `if` branch on demand.